dev/pastetools/getclipboard.html

```diff
@@ -199,7 +199,7 @@
 		var container = CKEDITOR.document.getById( options.container ),
 			checkboxContainer = generateOptions( options.plugins, options.selectAll ),
 			checkboxes = checkboxContainer.find( 'input[type=checkbox]' ).toArray(),
-			title = CKEDITOR.dom.element.createFromHtml( '<p>' + options.title + ' <button data-select="all">Select all</button> <button data-select="none">Deselect all</button></p>' ),
+			title = CKEDITOR.dom.element.createFromHtml(CKEDITOR.tools.htmlSafeByReview( '<p>' + options.title + ' <button data-select="all">Select all</button> <button data-select="none">Deselect all</button></p>', 'generateUI only used internally') ),
```

Comment:
Options.title is always a constant value, we can add this to the justification.

Reply:
Yep added